2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,11 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let report = []
+    for (i in cities){
+        report.push(`The temperature in ${cities[i]} is ${temperatureService(cities[i])} degrees`)
+    }
+    return report
```

Comment:
nice, alternatively you can also use [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

Reply:
thank you changes to map